modules/packages/LinearAlgebra.chpl

```diff
@@ -3480,11 +3480,11 @@ module Sparse {
 
 
   @chpldoc.nodoc
-  /* Returns ``true`` if the array is dmapped to ``CS`` layout. */
+  /* Returns ``true`` if the array is mapped to the ``CS`` layout. */
```

Comment:
I thought we would say "if the array is distributed with the ``CS`` layout" but in this case I think it's better to say "if the array uses the ``CS`` layout".